### PR TITLE
feat(hive): add support for hive metasore read/write

### PR DESCRIPTION
**Pull request by lyogev**

### CODE REVIEW

File: src/main/scala/com/yotpo/metorikku/metric/Metric.scala [hunk 1]
```diff
@@ -74,6 +74,36 @@ case class Metric(configuration: Configuration, metricDir: File, metricName: Str
     }
   }
 
+  private def writeToHive(job: Job, writer: Writer, outputConfig: Output) = {
```
Comment (RonBarabash) [inline src/main/scala/com/yotpo/metorikku/metric/Metric.scala]: why its not a part of the writer factory? why we have write and write to hive methods?

Comment (lyogev) [reply src/main/scala/com/yotpo/metorikku/metric/Metric.scala]: It's not part of a single writer. any writer can support this feature. so it should be outside of the writer.
It falls under the metric concern in my opinion.

File: src/main/scala/com/yotpo/metorikku/metric/MetricSet.scala [hunk 2]
```diff
@@ -19,7 +19,7 @@ object MetricSet {
   }
 }
 
-class MetricSet(metricSet: String) {
+class MetricSet(metricSet: String, write: Boolean = true) {
```
Comment (lyogev) [inline src/main/scala/com/yotpo/metorikku/metric/MetricSet.scala]: smelly

**Comment by lyogev** [comment]
closes #124

**Pull request merged**